packages/camera/camera_web/example/pubspec.yaml

```diff
@@ -6,9 +6,15 @@ environment:
   flutter: ">=3.13.0"
 
 dependencies:
-  camera_platform_interface: ^2.1.0
+  camera_platform_interface: ^2.6.0
   camera_web:
+    # When depending on this package from a real application you should use:
+    #   camera_web: ^x.y.z
+    # See https://dart.dev/tools/pub/dependencies#version-constraints
+    # The example app is bundled with the plugin so we use a path dependency on
+    # the parent directory to use the current plugin's version.
     path: ../
+
```

Comment:
Was this part of the changes before and I missed it? Please don't add blank lines in the `dependencies` block.

Reply:
I have backup of reviewed branch and there are no differences with squashed.

ok, I process new suggestions...

Reply:
removed line